src/mail.sh

```diff
@@ -78,17 +78,39 @@ function mail_send()
 {
   local flag="$1"
   local recipients="${options_values['TO']}"
+  local extra_opts="${options_values['PASSTHROUGH']}"
   local cmd='git send-email'
 
   flag=${flag:-'SILENT'}
 
   [[ -n "$recipients" ]] && cmd+=" --to=$recipients"
-
-  cmd+=' @^'
+  [[ -n "$extra_opts" ]] && cmd+=" $extra_opts"
 
   cmd_manager "$flag" "$cmd"
 }
 
+# This function checks if any of the arguments in @args is a valid commit
+# reference
+#
+# @args: arguments to be processed
+#
+# Returns:
+# 0 is any of the arguments is a valid reference to a commit; 1 otherwise
+function check_revision()
+{
+  local args="$*"
+  local arg
+
+  ! is_inside_work_tree && return 0
```

Comment:
Can we use `[[]]` or an `if` condition? I think this line differs slightly from the standard used in the code.